build/localnet/Tiltfile

```diff
@@ -176,22 +158,25 @@ k8s_yaml(["manifests/cli-client.yaml"])
 k8s_resource('pocket-v1-cli-client', labels=['client'])
 k8s_yaml(['manifests/cluster-manager.yaml'])
 k8s_resource('pocket-v1-cluster-manager', labels=['cluster-manager'])
-k8s_yaml(
-    [
-        local(
-            "templates/v1-validator-template.sh %s"
-            % localnet_config["validators"]["count"],
-            quiet=True,
-        ),
-    ]
-)
-
 
-# Uncomment in order to expose postgres port to 5432 on the host machine in case the developer wants to connect to postgres directly
-# First port is the port on the host machine, second is the port on the container, so if you already have postgres running on the host machine, you can change the first port to something else
-# k8s_resource(new_name='postgres',
-#              workload="dependencies-postgresql",
-#              port_forwards="5432:5432")
+check_helm_dependencies_for_chart(root_dir + "/charts/pocket-validator")
+for x in range(localnet_config["validators"]["count"]):
+    validator_number = x + 1
+    formatted_validator_number = local('printf "%03d" ' + str(validator_number))
```

Comment:
Why not do  `str(num).zfill(3)`? or `4` depending on what you're trying to achieve

Reply:
@Olshansk haha, that is a good question. This is not Python, but a dialect of it called Starlark. It offers very limited features, though the syntax is similar. 
Here are possible string interpolation expressions: https://github.com/bazelbuild/starlark/blob/master/spec.md - I didn't find the one that fits the use case and used `printf` instead. :)